### DISCUSSION TITLE
ICA-AROMA for atlas-based connectivity measure

### DISCUSSION THREAD

**Discussion by jb21469** [HALFpipe/HALFpipe#755] [General] [answered]
Hello,

I am running HALFpipe (v.1.2.2.post1.dev769+g04eafef4) to process a set of multiband (accel factor = 3) rsfMRI data. I would like to derive an atlas-based connectivity matrix (e.g., using the power 2011 atlas). For denoising, I would like to employ ICA-AROMA and remove the signal from WM and CSF (similar to an approach employed by [Ciric et al 2017](https://www.sciencedirect.com/science/article/pii/S1053811917302288)). 

Can you clarify how this would be achieved in HALFpipe? For example, when specifying processing for atlas-based connectivity by default there is no smoothing applied. However, it is my understanding that for ICA-AROMA the data are smoothed FWHM 6mm. So, are the noise components derived from smoothed data and regressed from unsmoothed data? Or are the resulting atlas-based connectivity data smoothed?

Thank you,
Jenna

**Comment by jb21469** [comment]
Hello,

Just checking back in on this. 

Thank you in advance!

Best,
Jenna

**Comment by HippocampusGirl** [comment, marked as answer]
Hi Jenna,
We accounted for this problem in HALFpipe in the following way:
The data will be smoothed and run through ICA-AROMA to generate component time series, which are classified into signal and noise. The noise time series are then regressed out of the unsmoothed data to generate the atlas-based connectivity matrices.

Best,
Lea
